test/runtests.jl

```diff
@@ -106,11 +106,11 @@ else
     t = digital_output(dev*"/Port0/Line0")
     @test typeof(t) == NIDAQ.DOTask
     @test start(t) == nothing
-    @test write(t, round(UInt32, [1,0,1,0,1,0])) == 6
+    @test @compat write(t, round.(UInt32, [1,0,1,0,1,0])) == 6
```

Comment:
this fails for me in 0.5.  moving the `@compat` to just before the `round` didn't help.  there's probably a way to rewrite this line that will work for both 0.5 and 0.6.  but i'd say retaining support for 0.5 is not worth the hassle of figuring it out.  if you agree, then just remove all references to compat and bump REQUIRE to 0.6.